src/app/clusters/resource-monitoring-server/resource-monitoring-cluster-objects.h

```diff
@@ -0,0 +1,211 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/CommandHandlerInterface.h>
+#include <app/util/util.h>
+#include <cstdint>
+#include <utility>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace ResourceMonitoring {
+
+static constexpr std::array<ClusterId, 12> AliasedClusters = { 0x0071, 0x0072, 0x0073, 0x0074, 0x0075, 0x0076,
+                                                               0x0077, 0x0078, 0x0079, 0x007a, 0x007b, 0x007c };
```

Comment:
This should just have the two clusters that are left, no?

Reply:
good catch, updated